.clang-tidy

```diff
@@ -34,6 +34,7 @@ Checks: "*,
 
         google-readability-avoid-underscore-in-googletest-name,
         google-upgrade-googletest-case,
+	      google-readability-braces-around-statements,
```

Comment:
I haven't checked `hicpp-braces-around-statements`, but `readability-braces-around-statements` seems to be slightly different than `google-readability-braces-around-statements`. `readability-braces-around-statements` seems to want braces around one-line if statements and the Google version hasn't complained about it at all. Does that change whether or not we want to include the other braces checks?

Reply:
There is a sub-option to set the number of lines and the Google version might have that turned on. If it's easy to make the other changes (maybe with -fix) then I think we should, otherwise I would just add whichever ones you've gotten to pass.